src/devices/SensorHub/tests/SensorHub_Not_Required.Tests.csproj

```diff
@@ -0,0 +1,22 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <!--
+  // NOTE: THIS TEST PROJECT IS NOT REQUIRED AND CAN BE DELETED IF NOT USED.
+  -->
+  
+  <PropertyGroup>
```

Comment:
If you're not using any unit tests, delete this file. 
Preferrable would be some tests, though.

Reply:
I don't think unit tests are that much useful if the device is mostly reading registers - sample project is more important IMO. Unless you mean to add unit tests using the device but then we need to make sure they won't be run by default

Reply:
True, there's not to much to test here, but when Mocking the i2cdevice, you could still test things like the combination of several registers into a single value.

Reply:
I added some basic tests for the registry reads